api/metrics-api-jaxrs/src/main/java/org/hawkular/metrics/api/jaxrs/MetricsServiceLifecycle.java

```diff
@@ -225,11 +228,10 @@ private void startMetricsService() {
             metricsService.setDateTimeService(createDateTimeService());
             metricsService.setDefaultTTL(getDefaultTTL());
 
-            // TODO Set up a managed metric registry
-            // We want a managed registry that can be shared by the JAX-RS endpoint and the core. Then we can expose
-            // the registered metrics in various ways such as new REST endpoints, JMX, or via different
-            // com.codahale.metrics.Reporter instances.
-            metricsService.startUp(session, keyspace, false, false, new MetricRegistry());
+            MetricRegistry metricRegistry = MetricRegistryProvider.INSTANCE.getMetricRegistry();
+            jmxReporter = JmxReporter.forRegistry(metricRegistry).inDomain("hawkular.metrics").build();
```

Comment:
I'd prefer not to add another config var unless it can be useful. Why would we want to have the Metrics collected but not accessible? It is very convenient to start jconsole or Mission control and have a a view of what's going on (threads, MBeans, memory, ... etc).